### DISCUSSION TITLE
Switch dashboard by sliding for touchscreens infotainment.

### DISCUSSION THREAD

**Discussion by Rozin14** [agronick/aa-torque#133] [General]
It seems that switching dashboards can be done only by dial and left right arrow in top right corner. Its hard to click on those arrows while driving, Sliding input is more suitable while driving. Requesting to add sliding input for touchscreens.

**Comment by agronick** [comment]
Just deployed this tonight. Buttons are gone. Swipe left and right or up and down.

**Reply by Southern-Discomfort** [reply]
Hey, did the swiping functionality go away? All was working fine until update to the beta. Now the swiping doesn't work even if you enable it. 
Just curious...
